.github/workflows/build-checking.yml

```diff
@@ -0,0 +1,42 @@
+name: Check Build
+
+env:
+  DRAGONFLY_PATH: "${{github.workspace}}/build-opt/dragonfly"
+  
+on: 
+  schedule:
+    - cron: '0 7 */3 * *'
+  workflow_dispatch:
+      
+jobs:
+  build-n-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: true
+      - name: Creating build
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y lsb-release ninja-build libunwind-dev libboost-fiber-dev libssl-dev autoconf-archive libtool cmake g++
+          ./helio/blaze.sh -release
+          cd build-opt && ninja dragonfly
+      - name: Run PyTests
+        run: |
+          chmod +x $DRAGONFLY_PATH
+          cd ${{ github.workspace }}/tests
+          pip install -r dragonfly/requirements.txt
+          pytest -xv dragonfly
+      - name: Send notification on failure
+        if: failure()
+        run: |
+          curl -s \
+            -X POST \
+            -H 'Content-Type: application/json' \
+            '${{ secrets.GSPACES_BOT_DF_BUILD }}' \
+            -d '{"text": "Unforunately the tests of dragonfly build are not passed :("}'
+      - uses: actions/upload-artifact@master
+        if: failure()
+        with:
+          name: build
```

Comment:
uploading a binary is redundant at this stage

Reply:
Okay, I will remove it